java/spacewalk-java.changes

```diff
@@ -1,5 +1,6 @@
 - Fix WebUI invalidation time by using the package build time instead
   of the WebUI version (bsc#1154868)
+- Tasks bunch detail page displays message "Task never started" in case a task get interrupted before start
```

Comment:
NitPick: Maybe make it a bit shorter. Moreover,  it doesn't hurt but please add your changelog at the very top, not in between.